<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupMembers</class>
 <widget class="QDialog" name="GroupMembers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>317</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Group Members</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>97</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="add_member_btn">
     <property name="text">
      <string>Add Member</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="remove_member_btn">
     <property name="text">
      <string>Remove Member</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="cancel_btn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>97</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QGroupBox" name="group_members_box">
     <property name="title">
      <string>Group Members</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="members_table">
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Role</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Submission Rate</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
